client/src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,37 @@
+import style from './styles.module.scss';
+import img1 from '@/assets/product_p.png';
+import button_up from '@/assets/button_up.svg';
+import button_down from '@/assets/button_down.svg';
+import { ProductsDescription } from './components/ProductDescription/ProductsDescribtion';
+import { BottomDescription } from './components/BottomDescription/BottomDescription';
+import { SectionTitle } from '@/shared/components/SectionTitle/SectionTitle';
+import { Products } from '../Products/Products';
```

Comment:
If Products component is not shared across the components it should be placed in 

ProductsCard/Products

now it is a standalone component